base/namedtuple.jl

```diff
@@ -226,6 +226,8 @@ end
 
 merge(a::NamedTuple{()}, b::NamedTuple) = b
 
+merge(a::NamedTuple, b::Iterators.Pairs{<:Any,<:Any,<:Any,<:NamedTuple}) = merge(a, b.data)
```

Comment:
Do we have an existing test that hits this code path?

Reply:
Yes, this is covered by the keywordargs tests.